### PR TITLE
Adds instructions for tests and switches to Slack for chat

### PR DESCRIPTION
**Pull request by aninternetof**
Apparently I missed committing some stuff to the README when I was setting up test.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,20 +40,27 @@ $ sudo ./start.sh #run this each time
...
 
 #### Development PC Alternate Setup (Docker)
-Rather use Docker? We have an image for you! First, on your development PC:
+Rather use Docker? First, on your development PC:
 ```bash
 $ sudo apt install git docker.io
 $ git clone --recursive https://github.com/aninternetof/rovercode.git && cd rovercode
-$ sudo docker pull cabarnes/rovercode
-$ sudo docker run --name rovercode -v $PWD:/var/www/rovercode -p 80:80 -d cabarnes/rovercode
+$ sudo docker build -t rovercode .
```
Comment (cabarnes) [inline README.md]: Do we still want to host an image on the Docker registry?

Comment (aninternetof) [inline README.md]: Yeah, I think so. I just didn't know if it was still getting updated with the move to store.docker, so I updated this to definitely work for now.

Comment (cabarnes) [inline README.md]: We can update this in the future if so

Comment (aninternetof) [inline README.md]: Sounds good.

**Pull request merged**